config/locales/en.yml

```diff
@@ -52,9 +52,9 @@ en:
         help: "You will consider candidates with a pending GCSE. But UCAS will block applications from someone who needs to take an equivalency test."
         details_label: "Taking"
       equivalence_test:
-        label: "3: Equivalence test"
+        label: "3: Equivalency test"
```

Comment:
Why is this change here?

Reply:
Paul G asked for it so done as a sperate commit